src/test/resources/connector/settings-zendesk4.json

```diff
@@ -1,6 +1,8 @@
 {
   "settings" : {
     "apiUrl" : "https://foo.com",
+    "zopimUrl" : "zopim.com",
```

Comment:
Does the case without these fields still work? Don't we want to test it as well?